src/hooks/useActiveAccount.tsx

```diff
@@ -23,15 +22,13 @@ export type ActiveAccountProps = {
 };
 
 export type ActiveAccountContextProps = ActiveAccountProps & {
-  refetch: () => Promise<QueryObserverResult<Account[], unknown>>;
-  setActiveAccountId: (accountId: string) => Promise<void>;
+  setActiveAccountId: (accountId: string) => void;
```

Comment:
There was no need for this function to be `async` -- it's just a wrapper around a `setState` call